internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -893,6 +925,106 @@ func (r *BMACReconciler) ensureSpokeMachine(ctx context.Context, log logrus.Fiel
 	return machineSpoke, nil
 }
 
+// Get MCS cert of the spoke cluster, encode it and add it to BMH
+func (r *BMACReconciler) ensureMCSCert(ctx context.Context, log logrus.FieldLogger, bmh *bmh_v1alpha1.BareMetalHost, agent *aiv1beta1.Agent) reconcileResult {
```

Comment:
we miss validation that mcs cert is already part of bmh. It looks like we will set it all the time and we will have reconciliation all the time

Reply:
Added a check here https://github.com/openshift/assisted-service/pull/1661/files#diff-86415614c6ed40c56c5ca2b5f751a65746ef5c83b61e6f021a4f5d817e4425baR950-R954